src/main/java/org/dependencytrack/model/Component.java

```diff
@@ -114,6 +115,11 @@ public enum FetchGroup {
     @Pattern(regexp = RegexSequence.Definition.PRINTABLE_CHARS, message = "The author may only contain printable characters")
     private String author;
 
+    @Persistent(defaultFetchGroup = "true")
+    @Convert(OrganizationalContactsJsonConverter.class)
+    @Column(name = "AUTHORS", jdbcType = "CLOB", allowsNull = "true")
+    private List<OrganizationalContact> authors;
```

Comment:
Keeping `author` and `authors` separate will introduce ambiguity to the data model. The CycloneDX specification had to do it this way to retain backward compatibility, but we should make an effort to keep our data model clean of such redundancies.

What I think we should do:

* When converting from CycloneDX to DT's data model, transform `author` to `authors`
* When converting from DT's data model CycloneDX, populate `author` with values from `authors`, **if targeting a CycloneDX version before 1.6** ([which we currently do](https://github.com/DependencyTrack/dependency-track/blob/af4461e86ae56eaee193017ec86ee2a690709932/src/main/java/org/dependencytrack/parser/cyclonedx/CycloneDXExporter.java#L97-L105))